pkg/monitoring/mimir/ingress/ingress.go

```diff
@@ -0,0 +1,40 @@
+package ingress
+
+import (
+	"os/exec"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/observability-operator/pkg/monitoring"
+)
+
+const (
+	secretName      = "mimir-gateway-ingress"
```

Comment:
You have this const twice right? How about adding it to pkg/monitoring/mimir/utils.go instead?

Reply:
I moved it to the utils file but I still have to declare those consts again because I can't use these constants in the ingress file since it's in a different package